src/App.tsx

```diff
@@ -1,35 +1,68 @@
 import { useState } from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import Registration from "./components/Registration";
-import Sign from "./components/Sign";
-import Panel from "./components/Panel";
+import Registration from "./pages/Registration";
+import Sign from "./pages/Sign";
 import EditPage from "./pages/EditPage";
 import PostEdit from "./pages/PostEdit";
-import Dialog from "./components/Dialog";
+import Users from "./pages/Users";
+import Posts from "./pages/Posts";
+import Dashboard from "./pages/Dashboard";
+
+interface User {
+  id: number;
+  fullname: string;
+  country: string;
+  number: string;
+  email: string;
+  gender: string;
+}
+
+interface Post {
+  id: number;
+  title: string;
+  description: string;
+  date: string;
+  user: string;
+}
+
 function App() {
   const [active, setActive] = useState(false);
+  const [confirmActive, setConfirmActive] = useState(false);
 
   const handleChangeActive = () => {
     setActive((previousEye) => !previousEye);
   };
 
+  const handleConfirmChangeActive = () => {
+    setConfirmActive((previousConfirmEye) => !previousConfirmEye);
+  };
+
+  const [users] = useState<User[]>([]);
+
+  const [posts] = useState<Post[]>([]);
+
   return (
     <div className="App">
       <BrowserRouter>
         <Routes>
-          <Route path="/" element={<Panel />} />
```

Comment:
aici se poate de facut in modul urmator:

in fisier routers.ts
```export const routes: Route[] = [
{ name: 'Dashboard', path: '/', exact: true, Component: Dashboard },
];
```

iar in ```App.tsx``` sa pui in modul urmator:

```
{routes.map(({ Component, ...item }) => (
              <Route key={item.name} name={item.name} path={item.path} exact={item.exact}>
                <Component />
              </Route>
            ))}```